web/src/stream_edit.ts

```diff
@@ -288,6 +288,13 @@ export function show_settings_for(node: HTMLElement): void {
             $(e.target).parent().remove();
         },
     );
+
+    $edit_container.find(".group-setting-label .help_link_widget i").on("click", (e) => {
+        // Clicking on the "?" icon should not trigger the click handler
+        // to open the pill input typeahead. Typeahead only needs to be
+        // opened when clicking on the label.
+        e.stopPropagation();
```

Comment:
It would be good to have this for realm and group settings as well if we add links for those settings in future.

One way can be to just have this handler in `enable_opening_typeahead_on_clicking_label` but that function is called during live-updating where we do not need this handler again.

Reply:
We are fairly likely to add `?` widgets for those in the future. It might be better to just make the typeahead opening logic have a check for whether it's inside a `.help_link_widget`, and return early there, rather than adding a handler that does `preventDefault`? We use that technique in `click_handlers.ts` for the main message feed in various places.